### PR TITLE
Set compatibility features on when importing from mqTranslate

### PR DESCRIPTION
**Pull request by xhaleera**
It should prevent many bugs for some users.

### CODE REVIEW

**Comment by mweimerskirch** [comment]
IMHO this should be set for all imports, not just for mq.

**Comment by xhaleera** [comment]
Other forks of the plugin do not use qtrans_ prefixed functions. qTranslate Plus, for example, use something like pptrans_

So, there is two options:
1. Only activated when importing from mqTranslate
2. Always enabled by default (then original qTranslate is supported also).

**Comment by mweimerskirch** [comment]
I didn't know that. I assumed they also had some sort of compatibility layer. If that is not the case, then yes, +1 from me.

File: qtranslate_frontend.php [hunk 1]
```diff
@@ -330,7 +330,7 @@ function qtranxf_excludeUntranslatedPosts($where,&$query) {//WP_Query
 	//qtranxf_dbg_echo('qtranxf_excludeUntranslatedPosts: post_type is empty: $query: ',$query, true);
 	//qtranxf_dbg_echo('qtranxf_excludeUntranslatedPosts: $where: ',$where);
 	//qtranxf_dbg_echo('qtranxf_excludeUntranslatedPosts: is_singular(): ',is_singular());
-	$single_post_query=is_singular();
+	$single_post_query=$query->is_singular();
```
Comment (johnclause) [inline qtranslate_frontend.php]: Is there a specific WP user, whose problem is solved by this change? Could you explain the logic why this is needed?

Comment (xhaleera) [inline qtranslate_frontend.php]: Sorry this commit is not related.
However, it solved a specific issue I had recently with mqTranslate.
Take for an example a page with a custom template inside which you set a second query using WP_Query. The is_singular() function is related to the main query recovering the page and not the WP_Query one. By doing this, is_singular() is always called for the good query.
